qt/aqt/utils.py

```diff
@@ -70,6 +120,107 @@ def openLink(link: str | QUrl) -> None:
         QDesktopServices.openUrl(QUrl(link))
 
 
+class MessageBox(QMessageBox):
+    def __init__(
+        self,
+        text: str,
+        callback: Callable[[str], None] = None,
```

Comment:
fyi, these type hints should include the `None` default value - in this case it should be:
`callback: Callable[[str], None] | None = None, `

Reply:
No problem, thanks for the review. I've added them.

Personally I think explicit Optionals are redundant, but I should have stayed consistent with the rest of the codebase.

Reply:
Currently mypy doesn't complain if they're omitted because of the `no_strict_optional = true` in mypy.ini, but once that is changed in the future, omitting them would be an error.